ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -187,52 +187,60 @@ tests:
   container:
     from: src
 - as: e2e-aws
-  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(azure|baremetal|gcp|libvirt|openstack|ovirt|vsphere)/)
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|azure|baremetal|gcp|ibmcloud|libvirt|openstack|ovirt|vsphere)/)
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
 - as: e2e-aws-proxy
   optional: true
+  skip_if_only_changed: (^(docs|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(alibabacloud|azure|baremetal|gcp|ibmcloud|libvirt|openstack|ovirt|vsphere)/)
```

Comment:
If we add `skip_if_only_changed` to a job that didn't previously have `always_run: true`, `skip_if_only_changed`, or `run_if_changed`, that means it will run automatically on many PRs where previously it was only for running manually. (This is extremely non-obvious.)
I think automatically starting jobs that were previously manual-only is out of scope for what you want to do in this change.

Reply:
That makes sense. I think we need to run some tests I think are important so ill add this to only those jobs(mostly upi versions of all cloud platform)

Reply:
OK. I would be inclined to make that a separate PR (or at least commit), but it's reasonable that the installer team might want that.
Just be aware that in the terms @patrickdillon put it earlier, this is effectively moving these jobs from "optional true, always_run false" to "optional true, always_run true".

Reply:
Fair point. Will wait for the team's thoughts on this.